stream-chat-swift-docs/docs/guides/working-with-user.md

```diff
@@ -1,3 +1,136 @@
 ---
 title: Working with User
----
\ No newline at end of file
+---
+
+The User object is accessed through a `Controller`. 
+There is a special type of user object which is `CurrentUser`, representing the currently signed-in user. 
+
+```swift
+/// User id of the user you want to work with
+let userId = "yourUserId"
+
+/// User controller for the intended user
+let userController = chatClient.usersController(userId: userId)
+
+/// Current user controller
+let currentUserController = chatClient.currentUserController()
+```
+
+## Standard User
+
+`CharUser` is a simple object, you can access the information and (un)mute and (un)flag the user.
```

Comment:
To introduce this entity smoother, we could slightly change the guide beginning to something like
"Stream Chat users are represented by `ChartUser` objects"

```suggestion
`ChartUser` is a simple object, you can access the information and (un)mute and (un)flag the user.
```

Reply:
did you mean `ChatUser`? 😅